euth/ideas/templates/euth_ideas/idea_list.html

```diff
@@ -37,5 +37,53 @@
     {% include "euth_ideas/includes/idea_list_tile.html" with idea=idea %}
     {% endfor %}
 
+    {% if is_paginated %}
+    <nav aria-label="Page navigation">
```

Comment:
Its a nav because it is a nav in Twitter Bootstrap: http://getbootstrap.com/components/#pagination